src/dispatch/incident/flows.py

```diff
@@ -1107,6 +1111,8 @@ def incident_engage_oncall_flow(
             incident_id=incident.id,
         )
 
+    return oncall_email
```

Comment:
Suggestion: we could return the individual and oncall_service, so the message we create in `handle_engage_oncall_action` could say `You have successfully engage X from team Y`.